README.md

```diff
@@ -1,2 +1,137 @@
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Html
+- CSS
+- Javascript
+
+## 🚀 Project Video <a name="live-video"></a>
+
+- [Project Video Link](https://www.loom.com/share/44ba81f11c0f4a93b1c03d0888b74f9f)
+Here you can see me explaining about my project.
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://supreeti.github.io/MYCAPSTONE/)
+  Here you can visit my project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git@github.com:supreeti/MYCAPSTONE.git
+```
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Supreeti**
+
+- GitHub: [@supreeti](https://github.com/supreeti)
+- Twitter: [@Ksupreeti](https://twitter.com/Ksupreeti)
+- LinkedIn: [Supreeti](https://linkedin.com/in/Supreeti)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!--  FEATURES -->
+
+## 🔭  Key Features <a name="features"></a>
+
+- [ 1] **[Home page]**
+- [ 2] **[About page]**
+- [ 3] **[Menu Popup using Javascript]**
+- [ 4] **[Responsive Website]**
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ 1] **[Registration page]**
+- [ 2] **[Login page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support by giving me ratings <a name="support"></a>
+
+If you like this project please contact author...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to give credit to <h4 color: blue>Cindy Shin </h4>the author of the original design in Behance (https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as required 
+
+I would also thank to Microverse and coding partners for support..
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License
+
+This project is [MIT](https://api.github.com/licenses/mit) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [optional] I will suggest that you use this [MIT license template](https://choosealicense.com/licenses/mit/) and personalize it, you did not filter and personalize the license you added to your README file.

- [optional] Please kindly remove this portion text for your README file to look professional.